packages/creator/src/helpers/add-content-to-solution.ts

```diff
@@ -203,14 +203,22 @@ export function _postProcessGroupDependencies(
           templates,
           dependencyId
         );
-        const gIndex = dependantTemplate.dependencies.indexOf(id);
-        /* istanbul ignore else */
-        if (gIndex > -1) {
-          removeDependencies = true;
-        }
-        /* istanbul ignore else */
-        if (dependantTemplate.groups.indexOf(id) < 0) {
-          dependantTemplate.groups.push(id);
+        // Not all items shared to the group will exist in the templates array
+        // i.e. Hub Initiative items or any other unsupported types
+        if (dependantTemplate) {
+          const gIndex = getWithDefault(
+            dependantTemplate,
+            "dependencies",
+            []
+          ).indexOf(id);
+          /* istanbul ignore else */
+          if (gIndex > -1) {
+            removeDependencies = true;
+          }
+          /* istanbul ignore else */
+          if (dependantTemplate.groups.indexOf(id) < 0) {
```

Comment:
@dbouwman groups are always good in this type of case? and shouldn't do something similar like:

dependantTemplate?.groups.indexOf(id) < 0

Reply:
Good question @jmhauck - in theory - `IItemTemplate` has a `.group` property, so it should be fine, but I'll add `dependantTemplate.groups?.indexOf(..)` just to cover us...

Reply:
Ok - I swapped this around to be even more defensive... so if `.groups` does not exist, it will add it to the `dependantTemplate`... not sure this is possible in the wild, but... I'm a fan of things not being breakable :)